### PR TITLE
add project toml

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name = "StatsMakie"
+uuid = "65254759-4cff-5aa5-8326-61ce017a8c70"
+version = "0.0.1"
+
+[deps]
+AbstractPlotting = "537997a7-5e4e-5d89-9595-2241ea00577e"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FreqTables = "da1fdf0e-e0ff-5433-a45f-9bb5ff651cb1"
+IndexedTables = "6deec6e2-d858-57c5-ab9b-e6ca5bd20e43"
```
Comment (piever) [inline Project.toml]: This is actually no longer needed. Instead I use `StructArrays` for the grouping (lowerbound at version 0.2.0)

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+
+[compat]
+FreqTables = ">=0.3.1"
+
+[extras]
+FreqTables = "da1fdf0e-e0ff-5433-a45f-9bb5ff651cb1"
```
Comment (piever) [inline Project.toml]: Maybe no longer needed as this is a normal dependency?

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+KernelDensity = "5ab0869b-81aa-558d-bb23-cbf5423bbe9b"
+Loess = "4345ca2d-374a-55d4-8d30-97f9976e7612"
+NamedArrays = "86f7a689-2022-50b4-a561-43c23ac3c673"
+Observables = "510215fc-4207-5dde-b226-833fc4488ee2"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+
+[compat]
```
Comment (piever) [inline Project.toml]: Should we also somehow say that the package relies on Julia 1.0? I'm not sure how one specifies this in a Project.toml file

**Comment by piever** [comment]
Thanks! I left a couple of comments

File: Project.toml (hunk 2 above)
Comment (SimonDanisch) [reply Project.toml]: this is generated automatically, so if you have changed stuff, maybe we should just rerun it!

File: Project.toml (hunk 3 above)
Comment (SimonDanisch) [reply Project.toml]: it seems like the conversion script simply doesn't catch that yet!

**Comment by piever** [comment]
Superseded by #64

**Pull request closed without merging**